AirCasting/SessionViews/SessionCartViewModel/MeasurementsViewModel.swift

```diff
@@ -0,0 +1,60 @@
+// Created by Lunar on 11/09/2021.
+//
+
+import Foundation
+import CoreLocation
+import CoreData
+
+class ViewModel: NSObject, ObservableObject {
+    
+    @Published var showLoadingIndicator = false
+    let sessionDownloader = SessionDownloadService(client: URLSession.shared,
+                                                   authorization: UserAuthenticationSession(),
+                                                   responseValidator: DefaultHTTPResponseValidator())
+    var measurementStreamStorage: MeasurementStreamStorage?
+    var task: Cancellable?
```

Comment:
I think this stuff can be private? And shouldn't they be injected using `init`?

I feel that only the `func syncMeasurements` and `showLoadingIndicator` should be public here to avoid confusion